src/tests/suite/SConscript

```diff
@@ -13,6 +13,15 @@ def scons():
     dfusetest = dfuse_env.d_program(File("dfuse_test.c"), LIBS='cmocka')
     denv.Install('$PREFIX/bin/', dfusetest)
 
+    shm_test_env = base_env.Clone()
+    shm_test_env.compiler_setup()
+    shm_test_env.AppendUnique(LIBPATH=[Dir('../../gurt')])
+    shm_test_env.AppendUnique(LIBPATH=[Dir('../../common')])
+    shm_test_env.AppendUnique(LIBPATH=[Dir('../../cart')])
+    shmtest = shm_test_env.d_program(File("shm_test.c"), LIBS=['gurt', 'daos_common', 'cart',
+                                     'cmocka', 'rt', 'pthread'])
+    denv.Install('$PREFIX/bin/', shmtest)
+
```

Comment:
i don't think you need a DAOS server to run those tests, right?
so probably adding those as unit tests in gurt will be more appropriate

Reply:
Right. Current tests are quite simple. DAOS server is not needed. I will change the tests as unit tests. We will add ftest later when necessary. Thank you!